src/js/hca-rjsf/config/form.js

```diff
@@ -17,11 +32,86 @@ const formConfig = {
       title: 'Veteran information',
       pages: {
         veteranInformation: {
-          path: 'veteran/information',
+          path: 'veteran-information/personal-information',
+          title: 'Veteran information',
+          initialData: {},
+          uiSchema: {
+            veteranFullName: _.merge(fullNameUISchema, {
+              first: {
+                'ui:errorMessages': {
+                  minLength: 'Please provide a valid name. Must be at least 2 characters.'
+                }
+              }
+            }),
+            mothersMaidenName: {
+              'ui:title': 'Mother’s maiden name'
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              veteranFullName: _.merge(fullName, {
+                properties: {
+                  suffix: {
+                    type: 'string'
+                  }
+                }
+              }),
+              mothersMaidenName: {
+                type: 'string'
+              }
+            }
+          }
+        },
+        birthInformation: {
+          path: 'veteran-information/birth-information',
+          title: 'Veteran information',
+          initialData: {},
+          uiSchema: {
+            veteranDateOfBirth: {
+              'ui:title': 'Date of birth'
+            },
+            veteranSocialSecurityNumber: ssn.uiSchema,
+            cityOfBirth: {
+              'ui:title': 'City of birth'
+            },
+            stateOfBirth: {
+              'ui:title': 'State of birth'
+            }
+          },
+          schema: {
+            type: 'object',
+            required: ['veteranDateOfBirth', 'veteranSocialSecurityNumber'],
+            properties: {
+              veteranDateOfBirth: date,
+              veteranSocialSecurityNumber: ssn.schema,
+              cityOfBirth,
+              stateOfBirth: _.merge(stateOfBirth, {
+                type: 'string'
```

Comment:
We can remove this once this PR is merged: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/87